static/app/components/events/interfaces/frame/stacktraceLink.tsx

```diff
@@ -192,7 +192,12 @@ export function StacktraceLink({frame, event, line}: StacktraceLinkProps) {
   const {projects} = useProjects();
   const hasStacktraceLinkFeatureFlag =
     organization?.features?.includes('issue-details-stacktrace-link-in-frame') ?? false;
-  const [isQueryEnabled, setIsQueryEnabled] = useState(!hasStacktraceLinkFeatureFlag);
+  const hasGithubSourceLink =
+    event.platform === 'csharp' &&
+    (frame.sourceLink || '').startsWith('https://www.github.com/');
```

Comment:
Can `sourceLink` have links to other providers? Looks like this is the existing behavior, but I wonder why we limited it to just github.

Reply:
It can be other providers, most common are GitHub and GitLab though and since the link from `sourceLink` isn't really the link we need to point to a line number (it's the raw link), we special cased GitHub only to to convert the link and append the line number.

Reply:
Would be good to add a comment above this line with the reason

Reply:
> did we already generate the sourceLink with the appropriate url with github/gitlab and the line number?

That is done on the FE. The BE keeps `source_link` as per spec, in other words: I doesn't rewrite the raw URL to the code viewer where we can append line number through URL anchor.

If we can want/can do the same rewriting for gitlab, bitbucket, azure devops or whatever other git hosting services we support, that would be ideal. Originally we left it out to simplify/reduce risk, since we couldn't dogfood that and .NET itself and popular libraries are on GitHub.